mlflow/utils/rest_utils.py

```diff
@@ -13,6 +14,25 @@
 RESOURCE_DOES_NOT_EXIST = 'RESOURCE_DOES_NOT_EXIST'
 
 
+class RestException(MlflowException):
```

Comment:
Hm, maybe a merge conflict? Can we use the `RestException` in `mlflow.exceptions`?